courses/courses_es.json

```diff
@@ -0,0 +1 @@
Url":"https://raw.githubusercontent.com/themonkslab/dream_teamers_courses/main/dart/9.Functions/13_New_methods_part_two.md","author":"The Monkslab","published":"2023-02-22T10:14:04.921464"},{"path":"9.Functions_14_Functional_vs_Imperative_Programming","title":"14.Functional vs Imperative Programming","description":"","contentUrl":"https://raw.githubusercontent.com/themonkslab/dream_teamers_courses/main/dart/9.Functions/14_Functional_vs_Imperative_Programming.md","author":"The Monkslab","published":"2023-02-22T10:14:04.922749"}]},{"path":"10.Classes","title":"10.Classes","description":"","articles":[{"path":"10.Classes_1_Introducción","title":"1.Introducción","description":"","contentUrl":"https://raw.githubusercontent.com/themonkslab/dream_teamers_courses/main/dart/10.Classes/1_Introducción.md","author":"The Monkslab","published":"2023-02-22T10:14:04.936374"},{"path":"10.Classes_2_Introducción_a_Classes","title":"2.Introducción a Classes","description":"","contentUrl":"https://raw.githubusercontent.com/themonkslab/dream_teamers_courses/main/dart/10.Classes/2_Introducción_a_Classes.md","author":"The Monkslab","published":"2023-02-22T10:14:04.936324"},{"path":"10.Classes_3_Instance_methods","title":"3.Instance methods","description":"","contentUrl":"https://raw.githubusercontent.com/themonkslab/dream_teamers_courses/main/dart/10.Classes/3_Instance_methods.md","author":"The Monkslab","published":"2023-02-22T10:14:04.936550"},{"path":"10.Classes_4_Class_constructors","title":"4.Class constructors","description":"","contentUrl":"https://raw.githubusercontent.com/themonkslab/dream_teamers_courses/main/dart/10.Classes/4_Class_constructors.md","author":"The Monkslab","published":"2023-02-22T10:14:04.936456"},{"path":"10.Classes_5_Getters_y_setters","title":"5.Getters y setters","description":"","contentUrl":"https://raw.githubusercontent.com/themonkslab/dream_teamers_courses/main/dart/10.Classes/5_Getters_y_setters.md","author":"The Monkslab","published":"2023-02-22T10:14:04.936263"},{"path":"10.Classes_6_Más_sobre_variables_y_methods","title":"6.Más sobre variables y methods","description":"","contentUrl":"https://raw.githubusercontent.com/themonkslab/dream_teamers_courses/main/dart/10.Classes/6_Más_sobre_variables_y_methods.md","author":"The Monkslab","published":"2023-02-22T10:14:04.936506"}]},{"path":"11.Medium_Dart","title":"11.Medium Dart","description":"","articles":[{"path":"11.Medium_Dart_1_Introduction","title":"1.Introduction","description":"","contentUrl":"https://raw.githubusercontent.com/themonkslab/dream_teamers_courses/main/dart/11.Medium_Dart/1_Introduction.md","author":"The Monkslab","published":"2023-02-22T10:14:04.924915"},{"path":"11.Medium_Dart_4_Debugging_a_project","title":"4.Debugging a project","description":"","contentUrl":"https://raw.githubusercontent.com/themonkslab/dream_teamers_courses/main/dart/11.Medium_Dart/4_Debugging_a_project.md","author":"The Monkslab","published":"2023-02-22T10:14:04.925087"}]}]}],"folder":"dart"}]
```

Comment:
this is the result of the `json_generator.dart`. Use https://jsonformatter.org/ to prettify it.